frontend/packages/kubevirt-plugin/src/components/vm.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+
+import {
+  getName,
+  getNamespace,
+  getUid,
+  // VmStatus,
+  // getSimpleVmStatus,
+  // VM_SIMPLE_STATUS_ALL,
+  // VM_SIMPLE_STATUS_TO_TEXT,
+  //  getResource,
+  //  DASHES,
+} from 'kubevirt-web-ui-components';
+
+import { NamespaceModel } from '@console/internal/models';
+import { ListHeader, ColHead, List, ListPage, ResourceRow } from '@console/internal/components/factory';
+import { ResourceLink } from '@console/internal/components/utils';
+// import { actions } from '../../module/okdk8s';
+
+import {
+  VirtualMachineModel,
+  // VirtualMachineInstanceModel,
+  // VirtualMachineInstanceMigrationModel,
+} from '../models';
+
+// import { openCreateVmWizard } from '../modals/create-vm-modal';
+// import { menuActions } from './menu-actions';
+// import { WithResources } from '../utils/withResources';
+
+const mainRowSize = 'col-lg-4 col-md-4 col-sm-6 col-xs-6';
+const otherRowSize = 'col-lg-4 col-md-4 hidden-sm hidden-xs';
+
+const VMHeader = props => <ListHeader>
```

Comment:
Heads up that we're in the process of updating all tables to PF4 (#1465)

@priley86 FYI, more tables :)

Reply:
```suggestion
const VMHeader: React.FC = props => <ListHeader>
```